src/index.ts

```diff
@@ -25,7 +35,7 @@ export class FormStrategy<User> extends Strategy<
   async authenticate(
     request: Request,
     sessionStorage: SessionStorage,
-    options: AuthenticateOptions
+    options: FormAuthenticateOptions
```

Comment:
This type can't be changed because you don't call `authenticate` from the strategy manually, and the Authenticator will not see this new option as a valid type.

Reply:
Yeah that was my bad not thinking! 

I've just updated to follow the normal strategy option format, but put the option param after verify because it's not required